definition_validator/validator.cc

```diff
@@ -180,6 +180,14 @@ void validateOverriding(const core::GlobalState &gs, core::SymbolRef method) {
                 e.addErrorLine(overridenMethod.data(gs)->loc(), "defined here");
             }
         }
+        if (!method.data(gs)->isOverride() && method.data(gs)->hasSig() && overridenMethod.data(gs)->isOverridable() &&
```

Comment:
this code has a bunch of `if`s.
Can we write a test for each combination of values for it?

Reply:
The various combinations are tested in some of the included tests.